Makefile

```diff
@@ -195,12 +203,12 @@ benchmark-opening_hours.js:
 benchmark-opening_hours.min.js:
 
 # .PHONY: benchmark
-benchmark-%.js: build/%.js scripts/benchmark.js
-	$(NODEJS) scripts/benchmark.js "../$<"
+benchmark-%.js: build/%.js scripts/benchmark.mjs
+	$(NODEJS) scripts/benchmark.mjs "../$<"
 
 .PHONY: check-package.json
 check-package.json: package.json
-	./node_modules/package-json-validator/bin/pjv --warnings --recommendations --filename "$<"
+	./node_modules/package-json-validator/lib/bin/pjv.mjs --warnings --recommendations --filename "$<"
```

Comment:
That seems to require an update of package-json-validator. Want to include this in that commit (to make it an atomic upgrade)?